component/portal/src/main/java/org/exoplatform/portal/mop/rest/UserNodeRestEntity.java

```diff
@@ -0,0 +1,121 @@
+package org.exoplatform.portal.mop.rest;
```

Comment:
please add license headers for all new files

Reply:
ok EntityBuilder also